unittests/data/type_traits_castxml.hpp

```diff
@@ -195,8 +195,8 @@ namespace no{
     typedef std::set< std::string > string_set_type;
     typedef std::multimap< std::string, std::string > s2s_multimap_type;
     typedef detail::vertex vertex_type;
-    typedef detail::const_item const_item_t;
-    typedef detail::const_container const_container_t;
+    // typedef detail::const_item const_item_t;
```

Comment:
I just remembered a thing about this, reading the history of the test file.

In commit 27cf5bb0ecb3b2eca65a313b45cb5ad86655bf2e I split the test in 2 (type_traits_castxml.hpp and type_traits_gccxml.hpp).

Next commit 5cbfc6b9fced9c1634a9beedd3ad8446ea2a3fc1 is more interesting. I tried to fix some failing tests when adding support for castxml. Reading the commit message, I remember being quite confused about what to do there.

Duplicating the hpp files was not the cleanest solution, and I had not final solution/fix for the initial problem. pyggcxml 1.7.0 was the first version to support castxml, and it was a tremendous effort to make all the tests pass (it took me around 6 months ...). So there were maybe 2-3 places were I took some strong decisions about what to do in the tests.

So I think that your patch is trying to fix the problem I initially saw. The castxml hpp file can probably be dropped now.